shared/validation-schemas/src/adoption.ts

```diff
@@ -1,3 +1,4 @@
+import "fast-text-encoding";
```

Comment:
just out of curiosity, why do we need this?

Reply:
left a link in the PR description https://github.com/hapijs/joi/issues/2141

Reply:
WDYT about adding a small comment at the top for future reference?

Reply:
yep, sounds good.